src/commands/deploy/buildImageInAzure/BuildImageStep.ts

```diff
@@ -23,7 +23,9 @@ export class BuildImageStep extends AzureWizardExecuteStep<IBuildImageInAzureCon
             const image = outputImages[0];
             context.image = `${image.registry}/${image.repository}:${image.tag}`;
         } else {
-            throw new Error(localize('noImagesBuilt', 'Failed to build image.'));
+            const logSasUrl = (await context.client.runs.getLogSasUrl(context.resourceGroupName, context.registryName, nonNullValue(context.run.runId))).logLink;
+            const errorMessage = localize('noImagesBuilt', 'Failed to build image. View [logs]({0}) for more details.', logSasUrl);
```

Comment:
Should we use a button instead of a link? I don't feel strongly one way or the other but it's worth considering both.